x/blob/README.md

```diff
@@ -46,7 +46,7 @@ message Params {
 
 #### `GasPerBlobByte`
 
-`GasPerBlobByte` is the amount of gas that is consumed per byte of blob data
+In v1 and v2`GasPerBlobByte` is the amount of gas that is consumed per byte of blob data
```

Comment:
This is also true for v3 so maybe this should clarify that before it was a governance modifiable param and then in v3 onwards it is a versioned param.